tests/testenv/cli_test_env.go

```diff
@@ -208,41 +207,14 @@ func (e *CLITest) Run(t *testing.T, args ...string) (stdout, stderr []string, er
 	c := exec.Command(e.Exe, cmdArgs...)
 	c.Env = append(os.Environ(), e.environment...)
 
-	stderrPipe, err := c.StderrPipe()
-	if err != nil {
-		t.Fatalf("can't set up stderr pipe reader")
-	}
-
-	var errOut []byte
-
-	var wg sync.WaitGroup
-
-	var pipeErr error
-
-	wg.Add(1)
-
-	go func() {
-		defer wg.Done()
-
-		errOut, pipeErr = ioutil.ReadAll(stderrPipe)
-		if goerrors.Is(pipeErr, os.ErrClosed) {
-			t.Logf("ignoring os.ErrClosed on stderr")
-
-			pipeErr = nil
-		}
-	}()
+	errOut := &bytes.Buffer{}
+	c.Stderr = errOut
```

Comment:
nit: Why not simply use, and then refer to it as `c.Stderr` below?
```suggestion
	c.Stderr = &bytes.Buffer{}
```

Reply:
Problem is `c.Stderr` is an `io.Writer` so I'd need to type assert it back to a `*bytes.Buffer` to get its `String` method